### PR TITLE
Add it.mijorus.whisper

### PR DESCRIPTION
**Pull request by mijorus**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project.
- [ x I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: it.mijorus.whisper.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    "sdk" : "org.gnome.Sdk",
+    "command" : "whisper",
+    "finish-args" : [
+        "--share=ipc",
+        "--share=network",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--device=all",
```
Comment (hfiguiere) [inline it.mijorus.whisper.json:14]:
```suggestion
        "--device=all",
```

File: it.mijorus.whisper.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        "--filesystem=xdg-run/pipewire-0"
```
Comment (hfiguiere) [inline it.mijorus.whisper.json:15]: why do you need this one?

File: it.mijorus.whisper.json [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        "python3-requirements.json",
+        {
+            "name" : "whisper",
+            "builddir" : true,
```
Comment (hfiguiere) [inline it.mijorus.whisper.json:32]:
```suggestion
```

it's always default with meson

File: it.mijorus.whisper.json (hunk 2 above)
Comment (mijorus) [reply it.mijorus.whisper.json:14]: `  "--socket=fallback-x11",`  was added automatically by Builder
` "--device=all",` I need to control audio devices
`  "--filesystem=xdg-run/pipewire-0"` same

Comment (hfiguiere) [reply it.mijorus.whisper.json:14]: do you call pipewire directly or JACK?

Comment (mijorus) [reply it.mijorus.whisper.json:14]: i call pipewire directly, with `pw link`

**Comment by barthalion** [comment]
bot, build it.mijorus.whispe

**Comment by barthalion** [comment]
bot, build it.mijorus.whisper

**Comment by mijorus** [comment]
bot, build it.mijorus.whisper

**Comment by mijorus** [comment]
bot, build it.mijorus.whisper

**Pull request closed without merging**